lib/crypto/random.js

```diff
@@ -4,4 +4,32 @@
 
 'use strict'
 
-module.exports = require('../promise').promisify(require('crypto').randomBytes)
+const randomBytes = require('../promise').promisify(require('crypto').randomBytes)
+
+function random(bytes) {
+  if (Array.isArray(bytes)) {
```

Comment:
I love the implementation here but just want to note that the API would be, imho, a little bit nicer for callers if it took variadic arguments rather than *single-value-or-array*. I know we can't lean on the rest/spread operators yet but we could lean on `arguments` in the meantime, with a view to switching to `...bytes` when we ditch node 4?

But also I realise there's a strong element of personal preference to this, so feel free to ignore me if you disagree.